cupyx/scipy/sparse/linalg/_iterative.py

```diff
@@ -1,10 +1,10 @@
-import numpy
-
 import cupy
+import numpy
```

Comment:
Could you roll back this modification in accordance with rule H306?

```
Import statements must be organized into three parts: standard libraries, third-party libraries, and internal imports. [H306]
```
https://docs.cupy.dev/en/latest/contribution.html#coding-guidelines

Reply:
Fixed in d2f93976a56c33f8440c637a3cb568a34787bf8f .
(This modification was accidentally inserted by vscode)